### PR TITLE
PLT-4357 Adding stats to HA

### PR DESCRIPTION
**Pull request by coreyhulen**
#### Summary
Fixing the admin console to show the proper stats when in HA mode.

https://github.com/mattermost/enterprise/pull/70

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4357

#### Checklist
- [X] Added or updated unit tests (required for all new features)
- [X] Has enterprise changes (please link)

### CODE REVIEW

**Comment by jwilander** [comment]
I think you forgot to submit the enterprise PR

**Comment by coreyhulen** [comment]
Submitted as https://github.com/mattermost/enterprise/pull/70

File: api/admin.go [hunk 1]
```diff
@@ -399,9 +399,33 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
...
+
+			totalSockets := 0
+			totalMasterDb := 0
+			totalReadDb := 0
+
+			for _, stat := range stats {
+				totalSockets = totalSockets + stat.TotalWebsocketConnections
+				totalMasterDb = totalMasterDb + stat.TotalMasterDbConnections
+				totalReadDb = totalReadDb + stat.TotalReadDbConnections
+			}
```
Comment (jwilander) [inline api/admin.go]: Unless I'm reading this wrong, doesn't this miss adding the stats for the current server?

Comment (coreyhulen) [inline api/admin.go]: @jwilander fixed.

**Pull request merged**